core-build/gulp-core-build/src/tasks/JestReporter.ts

```diff
@@ -6,15 +11,50 @@ const DEFAULT_REPORTER: any = require('jest-cli/build/reporters/default_reporter
  * rush does not get confused.
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-class JestReporter extends (DEFAULT_REPORTER as { new (globalConfig: any): any }) {
+class JestReporter extends (DEFAULT_REPORTER as { new (globalConfig: Jest.GlobalConfig): any }) {
```

Comment:
Can we replace the other `any`?

Reply:
I'm unsure what we would replace it with though. There's no type available for the actual interface of the reporter that can be used, since they come with the jest types package